### PR TITLE
Remove reflect and add Register func for qdb

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

File: pkg/service/command.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package service
+
+import (
+	"fmt"
+	"strings"
+
+	redis "github.com/reborndb/go/redis/resp"
+)
+
+type CommandFunc func(c *conn, args [][]byte) (redis.Resp, error)
```
Comment (siddontang) [inline pkg/service/command.go]: conn is not exported, so it is not proper here for outer register. 

Maybe we can still use a Session interface{}, and I don't want outer to use conn directly.

File: pkg/service/conn.go [hunk 2]
```diff
@@ -280,5 +282,9 @@ func (c *conn) SetDB(db uint32) {
 }
 
 func (c *conn) Store() *store.Store {
-	return c.bl
+	return c.h.store
+}
+
+func (c *conn) Handler() *Handler {
+	return c.h
 }
```
Comment (siddontang) [inline pkg/service/conn.go]: Why export Handler ?

Comment (qiuyesuifeng) [inline pkg/service/conn.go]: sorry, forgot it, so we make hander package unexported?

**Comment by siddontang** [comment]
LGTM

**Pull request merged**